monkey/monkey_island/cc/environment/server_config_generator.py

```diff
@@ -1,7 +1,8 @@
 from pathlib import Path
 
+from monkey_island.cc.consts import DEFAULT_STANDARD_SERVER_CONFIG_PATH
+
 
 def create_default_config_file(path):
-    default_config_path = f"{path}.default"
```

Comment:
so now `server_config.json.default` file is no longer being used? Why I don't see it being deleted from the codebase?

Reply:
It's there for the convenience of developers. I can remove it or rename it to `server_config.json.develop`

Reply:
How is convenient? The purpose of this file was to contain default value of `server_config.json`. We needed this whole process of config file deployment in order not to have `server_config.json` in our git, but have it on runtime.

Reply:
> How is it convenient?

As a developer, doing `cp server_config.json.default server_config.json` in a new development environment is less error prone than editing the file by hand.

> The purpose of this file was to contain default value of `server_config.json.`

Default for whom? Naming it something like `.develop` or `.standard` is more useful than `default`.

> We needed this whole process of config file deployment in order not to have `server_config.json` in our git, but have it on runtime.

That functionality still works. It's just using `server_config.json.standard` by default if the specified `server_config.json` doesn't exist.

Reply:
> As a developer, doing cp server_config.json.default server_config.json in a new development environment is less error prone than editing the file by hand.

How it worked before was that `server_config.json` was automatically created, which is even less error prone.

> That functionality still works. It's just using server_config.json.standard by default if the specified server_config.json doesn't exist.

Then we're back at square one: why do we need `server_config.json.default` ? I think the downside of using standard config for development is that in our statistics we won't be able to distinguish our own monkey usage from client monkey usage, but that doesn't sound like a big problem. If we think this is a problem, we need our codebase to use **develop** config by default and change this behavior to "use **standard** instead" during release process.